picker-starter/src/picker.config.ts

```diff
@@ -0,0 +1,40 @@
+import { getProductFilters, queryCategories, queryProducts } from '@/data'
+import { defineConfig } from '@/core'
+import { StoryblokIcon } from './components'
+import { optionsSchema } from './options-schema'
+
+export default defineConfig((options) => {
+  return {
+    title: 'Picker Starter',
+    icon: StoryblokIcon,
+    options: {
+      example: {
```

Comment:
I still wonder about having `example` as a part of this config schema. It may increase the amount of mandatory work for users to start this project.

They can include the example options within the error message of `validate` function's return if they want.

Reply:
Hey @eunjae-lee 🙌 

I [adjusted](https://github.com/storyblok/field-type-examples/pull/84/commits/d1d1b62cd88cc3c2a2fae82f658c01022bda1552) this config schema to consider both `options` and `example` as optional but maybe it isn't enough, still. 
I may need to remove this example as you said, and let the developer concatenate the example in the `validate` error message if they want.

What do you think? Making it optional would be enough or should we try getting rid of all this logic?

Reply:
@demetriusfeijoo I think it's still an optional thing that users might need to learn about. It'd be nice if we begin with small API surface, by getting rid of all this. What do you think?

Reply:
Yeah, I agree, let's make it more simple 💯 

I made these two changes here:
[45835c3](https://github.com/storyblok/field-type-examples/pull/84/commits/45835c34f003749a457e481f3f7b5c2e1a01c17a) and [8a9c4e7](https://github.com/storyblok/field-type-examples/pull/84/commits/8a9c4e780544421a7a5c20da135875a2082c7939)

Let me know what you think 🙌